get-started/index.md

```diff
@@ -7,24 +7,49 @@ mapped_pages:
 ---
 # Get started
 
-## Overview
+## Overview [what-is-es]
+
+[{{es}}](https://github.com/elastic/elasticsearch) is a distributed search and analytics engine, scalable data store, and vector database built on Apache Lucene. It’s optimized for speed and relevance on production-scale workloads. Use {{es}} to search, index, store, and analyze data of all shapes and sizes in near real time. [{{kib}}](https://github.com/elastic/kibana) is the graphical user interface for {{es}}. It’s a powerful tool for visualizing and analyzing your data, and for managing and monitoring the Elastic Stack. 
 
 {{es}} is the heart of the [Elastic Stack](the-stack.md). Combined with {{kib}}, it powers these Elastic solutions and use cases:
 
-* [Observability](/solutions/observability.md)
-* [Security](/solutions/security.md)
-* [Search](/solutions/search.md)
+* **[Observability](/solutions/observability.md)**: Resolve problems with open, flexible, and unified observability powered by advanced machine learning and analytics.
+* **[Security](/solutions/security.md)**: Detect, investigate, and respond to threats with AI-driven security analytics to protect your organization at scale.
+* **[Search](/solutions/search.md)**: Build powerful search and RAG applications using Elasticsearch's vector database, AI toolkit, and advanced retrieval capabilities.
 
 :::{tip}
 Refer to our [customer success stories](https://www.elastic.co/customers/success-stories) for concrete examples of how Elastic is used in real-world scenarios.
 :::
 
+## Step 1: Choose your deployment type
```

Comment:
FYI we have a snippet for this that we are using in various spots, if wanna reuse:

https://github.com/elastic/docs-content/blob/main/deploy-manage/_snippets/deployment-options-overview.md